core/src/main/java/bio/terra/pearl/core/service/participant/EnrolleeRelationService.java

```diff
@@ -0,0 +1,37 @@
+package bio.terra.pearl.core.service.participant;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import bio.terra.pearl.core.dao.participant.EnrolleeRelationDao;
+import bio.terra.pearl.core.model.participant.Enrollee;
+import bio.terra.pearl.core.model.participant.EnrolleeRelation;
+import bio.terra.pearl.core.service.CrudService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EnrolleeRelationService extends CrudService<EnrolleeRelation, EnrolleeRelationDao> {
+    EnrolleeService enrolleeService;
+
+    public EnrolleeRelationService(EnrolleeRelationDao enrolleeRelationDao, EnrolleeService enrolleeService) {
+        super(enrolleeRelationDao);
+        this.enrolleeService = enrolleeService;
+    }
+
+    public List<EnrolleeRelation> findEnrolleeRelationsByParticipantUserId(UUID participantUserId) {
+        return dao.findByParticipantUserId(participantUserId);
+    }
+
+    public List<Enrollee> findGovernedEnrollees(UUID participantUserId) {
+        List<EnrolleeRelation> enrolleeRelationsList =
```

Comment:
in general, you don't need to add 'list' at the end of variable names unless there's a bunch of lists, maps, sets, and you need to distinguish them
```suggestion
        List<EnrolleeRelation> enrolleeRelations =
```